pkg/cmd/run/download/download.go

```diff
@@ -57,7 +57,10 @@ func NewCmdDownload(f *cmdutil.Factory, runF func(*DownloadOptions) error) *cobr
 		  # Download a specific artifact within a run
 		  $ gh run download <run-id> -n <name>
 
-		  # Download specific artifacts across all runs in a repository
+		  # Download the latest artifact with a specific name
+		  $ gh run download -n <name>
+
+		  # Download multiple latest artifacts by name
 		  $ gh run download -n <name1> -n <name2>
```

Comment:
@sato11 : Instead of changing the explanation here within the examples, **perhaps we should add more content to the `Long` section that explains more fully?**

```
		Long: heredoc.Doc(`
			Download artifacts generated by a GitHub Actions workflow run.

			The contents of each artifact will be extracted under separate directories based on
			the artifact name. If only a single artifact is specified, it will be extracted into
			the current directory.

			By default, this command downloads the latest artifact created and uploaded through
			GitHub Actions. Because workflows can deleted or overwrite artifacts, `<run-id>`
			must be used to select an artifact from a specific workflow run.
		`),
```

Reply:
It sounds great to me and I will be undoing the current changes and opting for your idea. Thanks for the suggestion 🙂

Reply:
Made the change, rebased and renamed the title. Will you take a look again? @andyfeller